queries.sql

```diff
@@ -224,3 +224,8 @@ FROM (
     LIMIT 1
 ) AS subquery
 JOIN species s ON subquery.species_id = s.id;
+
+
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+explain analyze SELECT * FROM visits where vet_id = 2;
```

Comment:
- [ ] 👍 Great job including the screenshots for the `vet_id` index, both before and after its creation. However, upon closer examination, it appears that the screenshot after the index was created does not utilize the index. In the output, there is no "Index scan," and you can see a "Seq scan" instead. 

| Your | Expected|
| --- | --- |
|  ![image](https://github.com/JavierAybar/Vet-Clinic-DataBase/assets/55840999/15d6a395-762f-4e3e-829b-c4e847f22bef)| ![image](https://github.com/JavierAybar/Vet-Clinic-DataBase/assets/55840999/415c27f3-21dd-4f8a-a70a-0991146f7033)|

- [ ] To ensure that the index is functioning as expected, kindly consider following the instructions by removing the indexes and adding them once again. Afterward, take screenshots of the `explain analyze` output one more time to validate the proper utilization of the index. This will help ensure that the index is working as intended. Keep up the good work! 👍🔍📸